examples/config/dialog_on_exit_shutdown.config

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+
+<!-- Window -->
+<window>
+  <width>1000</width>
+  <height>845</height>
+  <default_exit_action>SHUTDOWN_SERVER</default_exit_action>
+  <dialog_on_exit>
+    <show>true</show>
+    <prompt_text>really? žřčš</prompt_text>
+    <show_shutdown_button>true</show_shutdown_button>
+    <show_close_gui_button>true</show_close_gui_button>
+    <shutdown_button_text>Quit Server&amp;GUI</shutdown_button_text>
+    <close_gui_button_text>Quit GUI only</close_gui_button_text>
```

Comment:
I am the user :) I'm building a kiosk running SubT simulator and the expectation is that it will be used by people who have no idea what runs under the hood. So showing them "Quit Server & GUI" would be really weird and I'd like to put there a simple "Exit" or something similar (I'm hiding the "Close GUI only" button as I always want to shutdown both; and I don't dare running both using `ign gazebo` as the GUI segfaults and would shot down the server cancelling the progress in the simulation; instead, I use good old `roslaunch` to keep the GUI alive =) ).